src/lua/cURL/impl/cURL.lua

```diff
@@ -717,7 +718,7 @@ end
 end
 -------------------------------------------
 
-setmetatable(cURL, {__index = curl})
+setmetatable(cURL, {__index = curl, __type = "LCURL"})
```

Comment:
This is not a class but just module. I do not sure that it should have a type metamethod